posthog-react-native/src/frameworks/wix-navigation.ts

```diff
@@ -23,12 +23,23 @@ export const withReactNativeNavigation = (posthog: PostHog, options: PostHogAuto
       return
     }
 
-    posthog.capture('Application Opened')
+    const appProperties = posthog.getAppProperties()
+    const appBuild = appProperties.$app_build
+    const appVersion = appProperties.$app_version
+
+    // TODO: add missing initialUrl
+    posthog.capture('Application Opened', {
```

Comment:
All events already contain `$app_build` and `$app_version` but for some reason (maybe legacy props), all the other SDKs were sending `version` and `build` as well for the lifecycle events.

Reply:
An option is to remove `version` and `build` from all those events as well, the mobile dashboards just rely on `$app_version` anyway, not sure if there's any other feature I am unaware of.

Reply:
Will remove them in the next major https://github.com/PostHog/posthog-js-lite/issues/212